src/components/KRadio/KRadio.vue

```diff
@@ -99,7 +99,7 @@ const props = defineProps({
    * Sets whether or not radio is selected
    */
   modelValue: {
-    type: [String, Number, Boolean, Object],
+    type: [String, Number, Boolean, Object, null] as PropType<string | number | boolean | object | null>,
```

Comment:
Why null instead of undefined (which I believe is the default)?

Reply:
The admin api doesn't recognize `undefined`, it's got to be explicitly `null`. I'm following [this pattern](https://vuejs.org/guide/components/props.html#nullable-type). I know this is a bit opinionated but VFG can't use KRadio otherwise 🤔